python/felis/metadata.py

```diff
@@ -274,13 +278,21 @@ def build_column(self, column_obj: datamodel.Column) -> Column:
             column_obj.autoincrement if column_obj.autoincrement is not None else "auto"
         )
 
+        server_default: str | TextClause | None = str(value) if value is not None else None
+        if server_default in _VALID_SERVER_DEFAULTS:
+            # If the server default is a valid keyword, use it as is.
+            server_default = text(str(server_default))
```

Comment:
Hmm, sorry, I had to re-read this. Does this mean that integer default becomes string and is rendered as string (quoted)?

Reply:
> Hmm, sorry, I had to re-read this. Does this mean that integer default becomes string and is rendered as string (quoted)?

Yes, this is a bug. I'll fix it so that non-string quotes are used as literals and not quoted.